content/pt-br/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+título: Contêiner de aplicações
+id: app-container
+data: 2023-01-24
+full_link:
+short_description: >
+   Um contêiner usado para executar parte de uma carga de trabalho. Pode ser comparado com um contêiner init.
```

Comment:
Suggestion: use "contêiner de inicialização" for init container and add the original term in parentheses.

Reply:
done